src/activity_handlers.cpp

```diff
@@ -2320,11 +2320,11 @@ void activity_handlers::repair_item_finish( player_activity *act, player *p )
 
     // Valid Repeat choice and target, attempt repair.
     if( repeat != REPEAT_INIT && act->targets.size() >= 2 ) {
-        item_location &fix = act->targets[1];
+        auto &fix_location = act->targets[1];
```

Comment:
Please don't replace types with auto, I find that over time it makes the code harder to understand.
Unless you're replacing an obvious but unreadable type signature (e.g. iterators or function signatures) or explicitly using auto to make code generic, avoid auto.

Reply:
Understood! Fixed in next commit. Thank you for your review.